DEVELOPING.md

```diff
@@ -67,6 +67,15 @@ The output of `make help` has a bit more information.
  - You will need to have [mockgen](https://github.com/golang/mock) installed
    to generate new or updated testing mocks for interfaces.
 
+## Blocking Ambassador telemetry
+Telemetry from dev builds to Ambassador Labs can be disabled by having your os resolve the `metriton.datawire.io` to `127.0.0.1`.
```

Comment:
The term "dev builds" is a misleading given that the suggested approach actually disables metrion reporting on the host where it is applied. Perhaps just strike "dev builds" from the sentence altogether?